notebooks/emissions_units.txt

```diff
@@ -0,0 +1,58 @@
+yr = 1 * year
+kt = 1000 * t
+
+C = [carbon_emissions]
+CO2 = 12/44 * C = carbon_dioxide_emissions
+
+N = [nitrogen_emissions]
+N2O = 14/44 * N = nitrous_oxide_emissions
+NOx = 14/46 * N = nox_emissions = NOX
+N2ON = 14/28 * N = nitrous_oxide_farming_style_emissions
+NH3 = 14/17 * N = ammonia_emissions
+
+S = [sulfur_emissions]
+SO2 = 32/64 * S = sulfur_dioxide_emissions
+SOx = SO2 = SOX
+
+VOC = [VOC_emissions] = NMVOC = non_methane_volatile_organic_compouns_emissions
```

Comment:
```suggestion
VOC = [VOC_emissions] = NMVOC = non_methane_volatile_organic_compounds_emissions 
```

Reply:
can you suggest again? Good suggestion just in the wrong file now